Keval/leds

captainrex493 opened the pull request:
Tested and works on oppertunitree

mihir-jain committed:
Basic led control implementation

- Method to send command to luma drive
- Command to choose led colors based on alliance
- Doesn't work on the robot

mihir-jain committed:
this works

captainrex493 committed:
Sets LED stripes to the correct alliance color

- Adds the "t" command to set presets
- Rewrites AllianceColorCommand.java to use "t" command
- Adds/Cleans up java docs
- Removes debugging stuff

captainrex493 committed:
Merge branch 'master' into keval/leds

dpk2442 commented on code/src/main/java/frc/robot/RobotContainer.java:
```diff
@@ -40,14 +41,20 @@
     intake.setDefaultCommand(driveArmCommand);
     climb.setDefaultCommand(climbCommand);
 
+    System.out.println(ledSubsystem);
```
All of these debug prints should be removed now

captainrex493 replied:
removed debug

dpk2442 commented on code/src/main/java/frc/robot/subsystems/LedSubsystem.java:
```diff
@@ -0,0 +1,78 @@
+
+    @Override
+    public void setPreset(Alliance alliance) {
+        if (alliance == Alliance.Blue) {
```
This could be a switch statement

dpk2442 commented on code/src/main/java/frc/robot/subsystems/LedSubsystem.java:
```diff
@@ -0,0 +1,78 @@
+    }
+
+    @Override
+    public void periodic() {
```
If you're not going to implement any functionality there's no need to implement this override

captainrex493 replied:
removed

captainrex493 committed:
Suggested changes

- Remove debug information
- Switches to switch from if statements
- Removed periodic

captainrex493 commented:
@dpk2442 Is that good?

captainrex493 committed:
Merge remote-tracking branch 'origin/master' into keval/leds

noam987 reviewed (dismissed):
This Looks Fine, but lets merge this after we finish up with shooter. That should hopefully be today or tomorrow.

captainrex493 committed:
Merge remote-tracking branch 'origin/master' into keval/led
